src/sonic-yang-models/yang-models/sonic-vrf.yang

```diff
@@ -0,0 +1,49 @@
+module sonic-vrf {
+	namespace "http://github.com/Azure/sonic-vrf";
+	prefix vrf;
+	
+	import sonic-extension {
+		prefix sonic-ext;
+	}
+
+	organization
+		"SONiC";
+
+	contact
+		"SONiC";
+
+	description
+		"SONIC VRF";
+
+	revision 2021-03-30 {
+		description
+			"Initial revision.";
+	}
+
+	container sonic-vrf {
+		container VRF {
+			description "Vrf configuration.";			 
+
+			list VRF_LIST {
+				key "vrf_name";
```

Comment:
From now on, I feel, we must use 4 spaces. I think few files are still using tabs.

Reply:
In the next PR, in a single shot, we can convert from TAB --> SPACEs.

Reply:
Addressed. I've converted all tabs to spaces.